hal/src/lib.rs

```diff
@@ -64,8 +64,3 @@ pub use self::samd21::*;
 pub mod samd11;
 #[cfg(feature = "samd11c14a")]
 pub use self::samd11::*;
-
-#[cfg(all(feature = "usb", feature = "samd21"))]
```

Comment:
Looking for some :eyes: on this change, it appears to me that these are already re-exported by the family globs `samd21::*` etc